csrc/quantization/pt_binding.cpp

```diff
@@ -192,16 +192,16 @@ std::vector<at::Tensor> quantized_reduction(at::Tensor& input_vals,
                               .layout(at::kStrided)
                               .device(at::kCUDA)
                               .requires_grad(false);
-    const int scales_elems = (quantize::requires_offset(quant_type)) ? 2 : 1;
-    auto scales = torch::empty({out_groups, scales_elems}, scales_options);
+    const int64_t scales_elems = (quantize::requires_offset(quant_type)) ? 2 : 1;
+    auto scales = torch::empty({(int64_t)out_groups, scales_elems}, scales_options);
```

Comment:
@earonesty - could you give samples of the errors you were hitting that caused you to change these?

Reply:
And was this to build all ops other than those not suppored on Windows (aio)?

Reply:
yes, this was everything i needed.   it was mostly "template ambiguity" errors.   there are multiple torch::empty funcs in the header. i can remove them and get the exact errors.

Reply:
that being said, code should probably always use int64_t when interacting with torch tensors, as a general rule